CONTRIBUTING.md

```diff
@@ -126,26 +112,13 @@ your clone of the originating `gh-pages` branch.
 Lastly, published copies of all the lessons are available in the `gh-pages` branch of the originating
 repository for reference while revising.
 
-## Other Resources
-
-General discussion of [Software Carpentry][swc-site], [Data Carpentry][dc-site], and [Library Carpentry][lc-site]
-happens on the [discussion mailing list][discuss-list],
-which everyone is welcome to join.
-You can also [reach us by email][email].
-
-[email]: mailto:team@carpentries.org
-[dc-issues]: https://github.com/issues?q=user%3Adatacarpentry
-[dc-lessons]: http://datacarpentry.org/lessons/
-[dc-site]: http://datacarpentry.org/
-[discuss-list]: https://carpentries.topicbox.com/groups/discuss
+[email]: 
+[i-site]: https://intersect-training.org/
+[i-issues]: tbd
+[i-lessons]: tbd
```

Comment:
```suggestion
```

Reply:
I don't think we need these, and would rather just drop them than leave them here to confuse.

Reply:
Should we keep the "issues" one? We can point it to the issues in this repo. Or we can just delete them both.

Reply:
It's not referenced/linked right now so let's remove it and add things back in for each lesson where needed.